drivers/azure/azure.go

```diff
@@ -298,6 +302,16 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Usage:  "Tags to be applied to the Azure VM instance",
 			EnvVar: "AZURE_TAGS",
 		},
+		mcnflag.StringFlag{
```

Comment:
I know this is rancher-machine and not terraform, but just to ask 1) do we have Azure docs where we specify what fields are being supported? That these new ones can be added to? Does the driver give the ability to specify required/optional for a flag?

Reply:
Currently there is no implementation for indicating required versus optional flags. The handling of flags was implemented within machine prior to Rancher's fork of the program. The `--help` flag is used to get arguments for each driver, since this is a CLI utility this is the norm and a supporting documentation page would likely be redundant.

Reply:
Ahh ok, cool thank you